ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -491,6 +491,46 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
+- as: e2e-aws-ipi-ovn-hypershift-mce-guest-p2
```

Comment:
@LiangquanLi930 I suggest to change `guest` to `hosted`.
it is better to change it in all files of the PR.

Reply:
@kuiwang02 I checked again for the `guest` words.  step `cucushift-hypershift-extended-disable-guest` is used for other existing test jobs too.  So if he update all of guest here, it will involve other existing test job update that would make this pr complex.  
I suggest he submit another pr to update the `guest` to `hosted` together for all jobs before submit any new test pipelines.  WDYT ? Thanks

Reply:
OK to make new PR to change guest to hosted.